src/Workspaces/Remote/Core/ProjectSystem/ILanguageServerProjectSystemService.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.Remote.ProjectSystem;
+
+internal interface ILanguageServerProjectSystemService
```

Comment:
The .NET Project System won't be using this interface directly (since the point is to bypass the project system for legacy projects) so if it is going to be internal you will need to use InternalsVisibleTo to make it available to the solution loader. The solution loader will also need to take a dependency on this assembly, whether or not the interface is public.

Reply:
@tmeschter Which assembly is needed for IVT?